singer_sdk/_singerlib/messages.py

```diff
@@ -8,7 +8,7 @@
 from dataclasses import asdict, dataclass, field
 from datetime import datetime, timezone
 
-import simplejson as json
+from singer_sdk.helpers._util import serialize_json
```

Comment:
I think I'd rather avoid imports here from modules outside `singer_sdk._singerlib`. Do you think it'd make sense to move some (minimal) of the serialization logic to `_singerlib`. I hope I'm not contradicting anything I've said before about `_singerlib` 😅

Reply:
No, you are not contradicting anything you have said before.  I was just trying to figure out a one spot were the functions that deal with JSON serialization and de-serialization would live.  Then utilize those functions for all JSON serialization and de-serialization.  I thought doing this would make it easier to troubleshoot issues and swap out JSON engines.

I am open to any suggestions.

Reply:
Edger, Your, "Move base SerDer to _singerlib" commit answered all my questions. Thanks🙏😊